src/uikit.cs

```diff
@@ -5175,6 +5175,164 @@ public interface UIGestureRecognizerDelegate {
 		bool ShouldReceivePress (UIGestureRecognizer gestureRecognizer, UIPress press);
 	}
 
+	[iOS (10,0)]
+	[BaseType (typeof(NSObject))]
+	interface UIGraphicsRendererFormat : NSCopying
+	{
+		[Static]
+		[Export ("defaultFormat")]
+		UIGraphicsRendererFormat DefaultFormat ();
```

Comment:
It should it be a getter-only property (like our `SharedInstance` are) or be prefixed with a `Get` so there's a verb in the method name.